angular.json

```diff
@@ -213,13 +213,25 @@
           }
         },
         "lint": {
-          "builder": "@angular-devkit/build-angular:tslint",
+          "builder": "@angular-eslint/builder:lint",
           "options": {
-            "tsConfig": [
-              "projects/dynamicforms/tsconfig.lib.json",
-              "projects/dynamicforms/tsconfig.spec.json"
-            ],
-            "exclude": ["**/node_modules/**"]
+            "lintFilePatterns": [
```

Comment:
I know this is copied from Spartacus, but please double check if we don't need exclusions here and why.

Reply:
When I revert to exclude option, I am getting  an error while running `ng lint`:
<img width="452" alt="Screenshot 2021-12-06 at 11 08 35" src="https://user-images.githubusercontent.com/75082466/144827737-2638f7dc-54e1-4506-90e4-4f2b05c9d199.png">

This command should remove `tsConfig` and `exclude` flags and insert new `lintFilePatterns`:
`ng g @angular-eslint/schematics:convert-tslint-to-eslint {{YOUR_PROJECT_NAME_GOES_HERE}}`

Here you can find the same problem if I add exclude option:
[https://stackoverflow.com/questions/63843748/how-to-update-the-configuration-for-eslint-after-upgrade-angular-9-to-10](url)